core/rawdb/database.go

```diff
@@ -409,7 +408,8 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 		case bytes.HasPrefix(key, blockReceiptsPrefix) && len(key) == (len(blockReceiptsPrefix)+8+common.HashLength):
 			receipts.Add(size)
 		case bytes.HasPrefix(key, headerPrefix) && bytes.HasSuffix(key, headerTDSuffix):
-			tds.Add(size)
+			// The TD has been removed, but we still count it here
```

Comment:
IMO better leave the `tds.Add` too, so we actually count the data, not just ignore it. Otherwise the numbers won't match up